app/models/base.py

```diff
@@ -1,7 +1,23 @@
-from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import String, DateTime, MetaData
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+import pytz
 
+# 한국 시간대 설정
+kst = pytz.timezone("Asia/Seoul")
 
-# Base 클래스는 SQLAlchemy의 모든 ORM 모델 클래스가 상속받는 기본 클래스입니다.
-# 이 클래스는 데이터베이스 테이블과 매핑되는 ORM 모델을 정의하는 데 사용됩니다.
+
+# 3개 다 필요함 공통된 칼럼을 상속받는 Base
 class Base(DeclarativeBase):
-    pass  # 이 클래스 자체는 비어 있지만, 상속받는 클래스들이 테이블을 정의하게 됩니다.
+
+    id: Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid4
+    )
+
+    created_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=lambda: datetime.now(kst)
+    )
+    updated_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=lambda: datetime.now(kst)
```

Comment:
nullable 관련해서는 필요 없는지 잘 몰라서 여쭤봅니다!

@chaeengg @trivialcoding6 @Ryan-OH @trivialcoding6 

```suggestion
        DateTime(timezone=True), default=lambda: datetime.now(kst), nullable=False
```

Reply:
헌찬님이 주신 내용으로 수정했고 다시 커밋 올라갈 예정입니닷!
그 레코드가 업데이트 될때 자동으로 저장되는 방식이라 nullable옵션은 따로 추가 안해도 될것같아요